staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -208,6 +208,8 @@ func (f *fancyResponseWriterDelegator) Flush() {
 }
 
 func (f *fancyResponseWriterDelegator) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	// fake a response status before protocol switch happens
+	f.processCode(http.StatusSwitchingProtocols)
```

Comment:
It's audit only. We don't have a proper way to express the hijack state. So neither the new nor the old audit event was correct. No need to revert IMO.